tests/infra/ccf.py

```diff
@@ -596,12 +596,12 @@ def wait_for_node_commit_sync(self, consensus, timeout=3):
             commits = []
             for node in self.get_joined_nodes():
                 with node.node_client() as c:
-                    r = c.get("getCommit")
+                    r = c.get("commit")
                     commits.append(r.seqno)
             if [commits[0]] * len(commits) == commits:
                 break
             time.sleep(0.1)
-        # in pbft getCommit increments the commit version, so commits will not be the same
+        # in pbft the commit rpc increments the commit version, so commits will not be the same
```

Comment:
This isn't true now that we're calling `set_execute_locally` for `commit`. If the PBFT tests work, I guess we just remove this comment?

Reply:
I think we can even make the check the same as for Raft.

Reply:
Yep.